src/centralserver/admin-ui/src/views/InitialConfiguration/InitialConfiguration.vue

```diff
@@ -334,6 +337,13 @@ export default (
           400 === errorStatus && 'validation_failure' === error?.error?.code
         );
       }
+
+      function isWeakPinError(error: ErrorInfo) {
+        let errorStatus = error.status;
+        return (
+          400 === errorStatus && 'weak_pin' === error?.error?.code
```

Comment:
There are styling warnings in this file -> this should fit in one line
```
return 400 === errorStatus && 'weak_pin' === error?.error?.code
```

The file should be formatted using Prettier